sale_automatic_workflow_payment_mode/models/sale_workflow_process.py

```diff
@@ -0,0 +1,16 @@
+# © 2016 Camptocamp SA, Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class SaleWorkflowProcess(models.Model):
+    _inherit = "sale.workflow.process"
+
+    @api.model
+    def _default_payment_filter_id(self):
```

Comment:
Is this function supposed to be called somewhere?

AFAICS there's a default value defined on payment_filter_id: https://github.com/OCA/sale-workflow/blob/14.0/sale_automatic_workflow/models/sale_workflow_process.py#L126

Which means, this will _fortunately_ never be called. :sweat_smile: